bletl/growth.py

```diff
@@ -176,13 +169,12 @@ def _make_random_walk(
     ----------
     name : str
         Name of the random walk variable.
+    init_dist
+        A random variable to use as the prior for innovations.
     mu : float, array-like
         Mean of the random walk.
-        If a vector is passed, only the first element should be nonzero,
```

Comment:
Why did you delete those lines?

Reply:
Previously this was just to get the first element of the random walk to have a different prior.

With the new `pm.RandomWalk` API one can have a completely independent distribution for the first element - the `init_dist`.
Here I used `pm.Normal`, but in principle one could specify a `pm.Exponential` for where a random walk initializes.